.github/workflows/run_psi4_test.yml

```diff
@@ -29,42 +26,36 @@ jobs:
     - name: Install psi4
       shell: bash -el {0}
       run: |
-        conda create -n p4env psi4 -c psi4
+        conda install psi4 python=3.8 -c conda-forge/label/libint_dev -c conda-forge
```

Comment:
also meh, python 3.8 hardcoded. Let's revert back to the general ubuntu image we had, and we'll try to be smart about windows and such later

Reply:
Psi4 only supports python 3.8 for Windows (native). Reverted